frontend/src/i18n/locales/de/glossary.json

```diff
@@ -0,0 +1,154 @@
+  "Daemon Sets": "Daemon Sets",
+  "Deployments": "Deployments",
+  "Replicas": "Replikate",
+  "Endpoint": "Endpunkt",
+  "Subsets": "Untergruppen",
+  "IP": "IP",
+  "Hostname": "Hostname",
+  "Target": "Ziel",
+  "Port": "Port",
+  "Protocol": "Protokoll",
+  "Endpoints": "Endpoints",
+  "Horizontal Pod Autoscalers": "Horizontal Pod Autoscalers",
+  "Rules": "Regeln",
+  "Host": "Host",
+  "Path": "Pfad",
+  "Service": "Service",
+  "Ingresses": "Ingresses",
+  "Hosts": "Hosts",
+  "Jobs": "Jobs",
+  "Completions": "Abschlüsse",
+  "Pod CIDR": "CIDR des Pods",
+  "Uptime": "Betriebszeit",
+  "System Info": "System-Informationen",
+  "Architecture": "Architektur",
+  "Boot ID": "Boot ID",
+  "System UUID": "UUID des Systems",
+  "OS": "OS",
+  "Kernel Version": "Kernel-Version",
+  "Machine ID": "Maschinen-ID",
+  "Kube Proxy Version": "Kube Proxy Version",
+  "Kubelet Version": "Kubelet-Version",
+  "Container Runtime Version": "Container-Laufzeitversion",
+  "Nodes": "Nodes",
+  "Logs: {{ itemName }}": "Protokolle: {{ itemName }}",
+  "Show Logs": "Protokolle anzeigen",
+  "logs": "Protokolle",
+  "Terminal / Exec": "Terminal / Exec",
+  "terminal": "Terminal",
+  "State": "Staat",
+  "Node": "Node",
+  "Host IP": "Host-IP",
+  "Pod IP": "Pod IP",
+  "Restarts": "Neustarts",
+  "{{ restarts }} ({{ abbrevTime }} ago)": "{{ restarts }} (vor {{ abbrevTime }})",
+  "Pod Disruption Budget": "Budget für Pod Disruption",
+  "Port Forwarding": "Portweiterleitung",
+  "PriorityClass": "PrioritätsKlasse",
+  "Replica Sets": "Replica Sets",
+  "Generation": "Generation",
+  "Resource": "Ressource",
+  "Resource Quotas": "Ressourcen-Quotas",
+  "Reference Kind": "Referenz Art",
+  "Reference Name": "Referenz Name",
+  "Binding Info": "Verbindliche Informationen",
+  "Role Bindings": "Role Bindings",
+  "API Groups": "API-Gruppen",
+  "Resources": "Ressourcen",
+  "Non Resources": "Nicht-Ressourcen",
+  "Verbs": "Verben",
+  "Roles": "Rollen",
+  "Secrets": "Secrets",
+  "Cluster IP": "Cluster-IP",
+  "External IP": "Externe IP",
+  "Services": "Services",
+  "Service Accounts": "Service Accounts",
+  "CRDs": "CRDs",
+  "Workloads": "Workloads",
+  "Stateful Sets": "Stateful Sets",
+  "CronJobs": "CronJobs",
+  "Storage Classes": "Storage Classes",
+  "Storage Volumes": "Speicher-Volumes",
+  "Persistent Volume Claims": "Persistent Volume Claims",
+  "Network": "Netzwerk",
+  "Security": "Sicherheit",
+  "Configuration": "Konfiguration",
+  "HPAs": "HPAs",
+  "Pod Disruption Budgets": "Budgets für Pod Disruption",
+  "Priority Classes": "Vorrangige Klassen",
+  "Git Version": "Git-Version",
+  "Git Commit": "Git Commit",
+  "Git Tree State": "Git-Tree-Status",
+  "Go Version": "Version starten",
+  "Platform": "Plattform",
+  "Kubernetes Version": "Kubernetes-Version",
+  "Capacity": "Kapazität",
+  "Access Modes": "Access Modes",
+  "Volume Mode": "Lautstärke-Modus",
+  "Storage Class": "Storage Class",
+  "Volume Claims": "Volumen Ansprüche",
```

Comment:
Doubt we'd translate Claims like this?

Reply:
I left it as "Volume Claims". Let me know if there's a better one for this.